.github/workflows/test-invoke-pip.yml

```diff
@@ -7,13 +7,11 @@ on:
       - 'pyproject.toml'
       - 'invokeai/**'
       - '!invokeai/frontend/web/**'
-      - 'invokeai/frontend/web/dist/**'
   pull_request:
     paths:
       - 'pyproject.toml'
       - 'invokeai/**'
       - '!invokeai/frontend/web/**'
```

Comment:
The invokeai/frontend/web/** can then be reoved as well (since we do not need a whitelist/blacklist there when we are not including dist in the tests.

Reply:
The web/ path is a negation/exclusion, so that changes to any thing in web/ are not tested. Pretty sure it's needed here... 

The issue was that changes to the built ui, in web/dist/, were triggering tests. Nothing in web/ should trigger tests, even web/dist/